cmd/kinflate/demos/README.md

```diff
@@ -0,0 +1,10 @@
+# Demos
+
+These demos are covered by presubmit tests.
+
```

Comment:
yes

Reply:
that's also why i took the demo list off the main page - so we'd only have one list to maintain.

Ahmet thinks every directory should have a README, so this will be a list of the demos with short descriptions of what they do.